File: app/src/main/java/com/rizafu/sample/SequenceFragment.java
package com.rizafu.sample;

import android.os.Bundle;
import android.support.v4.app.Fragment;

/**
 * Created by RizaFu on 2/27/17.
 */

public class SequenceFragment extends Fragment {

    public static SequenceFragment newInstance() {

        Bundle args = new Bundle();

        SequenceFragment fragment = new SequenceFragment();
        fragment.setArguments(args);
        return fragment;
    }

}

File: app/src/main/java/com/rizafu/sample/SequenceFragment.kt
package com.rizafu.sample

import android.os.Bundle
import android.support.v4.app.Fragment

/**
 * Created by RizaFu on 2/27/17.
 */

class SequenceFragment : Fragment() {
    companion object {

        fun newInstance(): SequenceFragment {

            val args = Bundle()

            val fragment = SequenceFragment()
            fragment.arguments = args
            return fragment
        }
    }

}
